remote/Dockerfile

```diff
@@ -27,7 +27,8 @@ RUN apt-get update && apt-get upgrade -y && apt-get install -y --no-install-reco
     obconf \
     gettext-base \
     xprintidle \
-    nitrogen && rm -rf /var/lib/apt/lists/*
+    && rm -rf /var/lib/apt/lists/* && \
+    sed -i -e "/^border\.width:/ s/: .*/: 3/" /usr/share/themes/Clearlooks/openbox-3/themerc
```

Comment:
This only seems to increase the bottom border size. This looks a bit odd to me since the borders on the left and right still have the previous size. Since the left and right borders can also be used for resizing, we should check if it's feasible to increase their size (probably via theming)